tests/compat.py

```diff
@@ -4,7 +4,7 @@
 try:
     import zipp  # nopycln: import
```

Comment:
While you are at it, you could rewrite it to `if sys.version < (3, x)`, that way it will be recognized by `pyupgrade` ([link](https://github.com/asottile/pyupgrade#python2-and-old-python3x-blocks)) and will be removed when we upgrade Python versions.

Reply:
Nice, I didn't know that `pyupgrade` handled that, thanks!